Archive 2020-01-08-governance.md (#181)
hyphacoop/organizing: 2020-01-08-governance.md

---
breaks:  false # See: https://github.com/hackmdio/codimd/issues/40#issuecomment-172927690
private: false # See: https://github.com/hyphacoop/hyphacoop-chatbot#archive
---
# 2020-01-08 Hypha Worker Co-op: Governance

<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>

Time: Wednesday, Jan 08, 2020 @ 3-4.30pm  
Location: https://jitsi.hypha.coop/hyphacoop  
Attending: pc, dw  
Cultivator: dw  
Notetaker: dw 

## Agenda

| Item                                        | Time (min) |
|:--------------------------------------------|-----------:|
| [Personal Checkins](#Personal-Checkins)     |          5 |
| [Milestones Review](#Milestones-Review)     |         20 |
| [Task Board Review](#Task-Board-Review)     |         20 |
| [Discussions](#Discussions)                 |         40 |
| [Process Checkout](#Process-Checkout)       |          5 |

## Notes

### Personal Checkins

> - Pre-convo took us into meeting

Reviewer: ```suggestion
- pc: pre-convo that took us into meeting
```

        - Defined “Local Solidarity and Decentralized Neighbours”
- Notes:
    - dc: "First directors selected" an outcome, not milestone?
    - pc: "first training"? --> aspirational from using our development fund!
    - pc: "operations implementation report"? dc: think this was a status update
    - dc: is engagement strategy mostly business development?

From 2019-12-18 all hands:  
- ✅ Tracked organizational actions via task tracker
- ✅ Conducted “informational interviews” with other co-operatives
- ✅ Drafted and submitted articles of incorporation
    - ✅ Determined head office
    - ✅ Determined “incorporators” and “directors”
- ✅ Submitted initial return
- ✅ Developed co-operative resolution and signing process
- ✅ Ensured compliance with WSIB expectations
- ✅ Set up time for 2020 Q1 regular meetings and co-work session to work on:
    - Draft initial bylaws
    - Plan for initial General Meeting 2020 Q1

### Task Board Review

- [Governance][gov-wg] :white_check_mark: [`wg:governance`][l-gov]
    - Archive _Done_ tasks :tada:
    - Review [HIGH `[priority-★★★]`][l-pri-hi]
    - Triage new tasks
    - missing labels [`wg:NULL`][l-none] | [`[priority-NULL]`][l-pri-none]

> - Did a triage in order except looked at missing labels 

Reviewer: ```suggestion
- Did triage in order except for looking at missing labels 
```

### Discussions

- Governance and Operations boundary?

>     - dc: concern over just bolting Ops on, should be its own thing (even if meets together/has same people for now.)

Reviewer: ```suggestion
    - dc: concern over just bolting Ops on, should be its own thing (even if it meets together/has same people for now)
```

        - very different scale of action

>         - worried if/how operations gets "ghettoized" as the no one wants to do tasks if left by itelf

Reviewer: ```suggestion
        - worried if/how operations gets "ghettoized" as the no one wants to do tasks if they are left by themselves
```

    - pc: importance of defining roles
- What makes a successful AGM? What is it not?
    - pc: Should feel celebratory. Assuming bare minimum requirements are already done. Shouldn't feel rote/procedural (despite things being stacked in favour of that happening)

>     - dc: Alleviates tensions from previous year and sets a course for upcoming, in motivating way. Not a place to initiate controversial convos. If challenging topics, should have prep. No surprises.

Reviewer: ```suggestion
    - dc: Alleviates tensions from previous year and sets a course for upcoming year in a motivating way. Ideally, not a place to initiate controversial convos. If challenging topics, should have prep. No surprises to most people there.
```

- Proposal for Operations:

>     - New tag for "in between" tasks (`needs-review`, `ownership-issue`, `boundary-object`)

Reviewer: ```suggestion
    - New tag for "in between" tasks (e.g., could be `needs-review`, `ownership-issue`, `boundary-object`)
```

    - `wg:ops` as new tag, rename existing to `wg:infra` 
    - "Ops" work (potentially) addressed at Governance WG for now.
- Things to take to meeting today:
    - Question: 
        - Are there roles that you think we need to be successful? Which ones do you think are most important?
        - What makes a successful AGM? What is it not?
    - Proposal for Operations

### Parked discussions

- #140 Improve org-wide task flow
- What do we want to get out of these meetings?
- #102 Roadmapping to the AGM
- Scope of Working Group
- Peer-reviewing
 
### Process Checkout

- Did not happen today

### Action Items for Cultivator

> - [ ] Migrate items with `#todo:person` to [task board][tasks]

Reviewer: ```suggestion
- [x] Migrate items with `#todo:person` to [task board][tasks]
```

> - [ ] Publish meeting notes to [GitHub][gh]

Reviewer: ```suggestion
- [x] Publish meeting notes to [GitHub][gh]
```
